components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -2944,10 +2944,10 @@ subroutine ocn_time_integration_split_init(domain)!{{{
 
       !-----------------------------------------------------------------
       if (config_use_self_attraction_loading) then
-         ! set ssh_sal_on to 1
-         ssh_sal_on = 1
+         ! set pgf_sal_on to 1
+         pgf_sal_on = 1
```

Comment:
Can we use this as an opportunity to make these 1.0_RKIND here for consistency? (Here and elsewhere in this file)